libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/binder/UIBinder.kt

```diff
@@ -18,6 +18,8 @@ interface Binder<T> {
      * [MapboxNavigationObserver] which gives this view a simple lifecycle.
      */
     fun bind(value: T): MapboxNavigationObserver
+
+    fun unbind(value: T) = Unit
```

Comment:
Curious on this one, why was this needed? The returned `MapboxNavigationObserver` calls `.onDetached`

Reply:
That's a fix for MapView being removed on detached.

Reply:
@tomaszrybakiewicz will you please revert the changes to `UIBinder` and `UICoordinator`

We will address the issue for removing the map separately. Doing this unbind impacts all UIBinder, but we have a special exception with the MapCoordinator.

Reply:
Done. I broke it again.